### ISSUE TITLE
[2.1 & 3.0]: PHP Fatal error:  Uncaught Error: Interface "MatthiasMullie\PathConverter\ConverterInterface" not found

### ISSUE THREAD

**Issue by sbulen** [SimpleMachines/SMF#8244]
### Basic Information

I get this error when attempting to run a profile export (PMs and posts, human readable).  

Error exists in both 2.1 & 3.0.

### Steps to reproduce

1.  Install fresh 2.1 with current GitHub as of the time of post
2.  Post a couple messages & PMs
3.  Attempt a profile export, PMs and posts, human readable

### Expected result

An exported profile.

### Actual result

The process hangs, leaving an entry in background tasks.  No errors in SMF, but the reported error is in the PHP error log.

### Version/Git revision

2.1.5, current GitHub

### Database Engine

MySQL

### Database Version

8.0.31

### PHP Version

8.2.19

### Logs

```shell
The error is NOT in the SMF log, it is in the PHP error log.  

[10-Jun-2024 18:43:05 UTC] PHP Fatal error:  Uncaught Error: Interface "MatthiasMullie\PathConverter\ConverterInterface" not found in D:\wamp64\www\80van21\Sources\minify\path-converter\src\Converter.php:19
Stack trace:
#0 D:\wamp64\www\80van21\Sources\Profile-Export.php(1818): include_once()
#1 D:\wamp64\www\80van21\Sources\Profile-Export.php(1678): export_load_css_js()
#2 D:\wamp64\www\80van21\Sources\tasks\ExportProfileData.php(621): get_xslt_stylesheet('XML_XSLT', 1)
#3 D:\wamp64\www\80van21\Sources\Subs.php(5773): ExportProfileData_Background::add_dtd(Array, Array, Array, Array, Array, Array, 'smf', 'profile', '')
#4 D:\wamp64\www\80van21\Sources\News.php(349): call_integration_hook('integrate_xml_d...', Array)
#5 D:\wamp64\www\80van21\Sources\tasks\ExportProfileData.php(251): buildXmlFeed('smf', Array, Array, 'profile')
#6 D:\wamp64\www\80van21\Sources\tasks\ExportProfileData.php(552): ExportProfileData_Background->exportXml(Array)
#7 D:\wamp64\www\80van21\Sources\tasks\ExportProfileData.php(120): ExportProfileData_Background->exportXmlXslt(Array)
#8 D:\wamp64\www\80van21\cron.php(249): ExportProfileData_Background->execute()
#9 D:\wamp64\www\80van21\cron.php(131): perform_task(Array)
#10 {main}
  thrown in D:\wamp64\www\80van21\Sources\minify\path-converter\src\Converter.php on line 19
```

### Additional Information

As reported in the forum:
https://www.simplemachines.org/community/index.php?topic=589098.0

**Comment by sbulen** [comment]
Note this related topic in the forum - "Use of "parent" in callables is deprecated":
https://www.simplemachines.org/community/index.php?topic=589127.0

Related recent activity:
#7725 
#7726

**Comment by sbulen** [comment]
FYI, I recommend this one for 2.1.5.

**Comment by sbulen** [comment]
This can be closed.

The 2.1 error was addressed by @Sesquipedalian with #8270 

The 3.0 one cannot be reproduced with current code anymore, so something fixed that along the line as well (maybe the tweaks to the autolinker logic???).  

While retesting, I found two unrelated errors with export.  Both are bonehead easy so I'lll just submit a PR.
